packages/react/src/shared-constants.ts

```diff
@@ -0,0 +1,2 @@
+export const SHOPIFY_Y = '_shopify_y';
+export const SHOPIFY_S = '_shopify_s';
```

Comment:
I simulated if I put these constants inside `analytics-constants.ts` file and remove any usages of analytics related functions within the nextjs app. The result would be that everything in the `analytics-constants.ts` will be packaged as part of the build.

<img width="825" alt="Screenshot 2023-01-17 at 10 48 28 AM" src="https://user-images.githubusercontent.com/2319002/212986465-6a0d79e3-467a-4362-aa1a-1cc2faed85f1.png">

This is what I am trying to avoid. While it seems very unorganized to have so many file group of constants, in build time, it makes sense to have such separation.

At the very least, I need to have `analytics-constants.ts` to be a separate constant file because these will grow over time. I don't want it to be part of a project build if analytics related functions are not being used.